mmcv/runner/ipu/hooks.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.utils import TORCH_VERSION, digit_version
+from mmcv.runner.hooks import HOOKS, OptimizerHook, LrUpdaterHook
+
+
+def wrap_lr_update_hook(lr_hook_class,):
```

Comment:
```suggestion
def wrap_lr_update_hook(lr_hook_class):
```

Reply:
Suggest adding a summary or description to explain why we need to wrap `lr_update_hook`, which will be helpful to users or other developers.